test/core/zapper/WidoZapperGammaRetro.t.sol

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.7;
+
+import "forge-std/Test.sol";
+import "forge-std/StdUtils.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "../../shared/PolygonForkTest.sol";
+import "../../../contracts/core/zapper/WidoZapperGammaRetro.sol";
+
+contract WidoZapperGamma_Retro_Test is PolygonForkTest {
```

Comment:
Can we also add tests for how much dust is returned to the user? Maybe have a percentage threshold on the dust returned?